fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/common/StoreTimer.java

```diff
@@ -167,6 +166,42 @@ protected static Counter getCounter(@Nonnull Map<Event, Counter> counters, @Nonn
          * @return the user-visible title
          */
         String title();
+
+        /**
+         * Get the key of this event for logging. This should be used with
+         * {@link KeyValueLogMessage}s and other key-value based logging
+         * systems to log the values from instrumented events. These keys are
+         * not expected to change frequently. They may, however, change
+         * outside of any minor version change. Their values, therefore,
+         * should not be relied upon, other that for the logging purposes.
+         *
+         * By convention, the following abbreviations are recommended:
```

Comment:
Needs a paragraph tag to be rendered correctly

Reply:
Removed the list of suggested abbreviations.